### PR TITLE
Fixed timer (hopefully? Skparab please check this)

### PR DESCRIPTION
**Pull request by apawate**
This might have fixed the error where the timer starts at a number above zero when the game starts.

### CODE REVIEW

**Comment by Skparab1** [comment]
ok ye colon may not work but fix shud fix problem

File: canvas-controller.js [hunk 1]
```diff
@@ -423,6 +424,9 @@
           }
 
           if (counter >= 1 && startwaiter){
+            if starting:
+                  start = Date.now();
```
Comment (phuang1024) [inline canvas-controller.js:428]: Pretty sure this is js not python

Comment (apawate) [reply canvas-controller.js:428]: yeah @Skparab1 fixed it

Comment (Skparab1) [reply canvas-controller.js:428]: ye the only problem was the colon, you have to use curly braces lol

**Pull request merged**